### ISSUE TITLE
About init networks and optimizer

### ISSUE THREAD

**Issue by kirk0221** [charlesCXK/TorchSemiSeg#89]
Dear author, i have some question about your work.
In your paper, networks have different their init weight.

> The two networks have the same structure and their weights, i.e., θ1 and θ2 , are initialized differently.

Then your code have same init weight(city8.res50v3+CPS, line 82~91)
`  
    # define and init the model

    model = Network(config.num_classes, criterion=criterion,
                    pretrained_model=config.pretrained_model,
                    norm_layer=BatchNorm2d)

    init_weight(model.branch1.business_layer, nn.init.kaiming_normal_,
                BatchNorm2d, config.bn_eps, config.bn_momentum,
                mode='fan_in', nonlinearity='relu')

    init_weight(model.branch2.business_layer, nn.init.kaiming_normal_,
                BatchNorm2d, config.bn_eps, config.bn_momentum,
                mode='fan_in', nonlinearity='relu')`

And how do you set the optimizer_l and optimizer_r in the branches of each model?
Can the optimizer be set on each branch without using the group_weight function?
